Sources/SwiftGen/Commander/ParserCLICommands.swift

```diff
@@ -9,38 +9,42 @@ import PathKit
 import StencilSwiftKit
 import SwiftGenKit
 
-// deprecated option
-private let deprecatedTemplateNameOption = Option<String>(
-  "template",
-  default: "",
-  flag: "t",
-  description: """
-    DEPRECATED, use `--templateName` instead
-    """
-)
+extension ParserCLI {
+  private enum CLIOption {
```

Comment:
I'd rename this to `Option` (we already know it's for "CLI" because it's inside `ParserCLI`)

Reply:
Problem is, renaming it `Option` conflicts with Commander's `Option` and confuse the compiler. We can go over it and use compiler hints or fully-qualified names for the type, but I thought choosing a different name was less complicated.